console-extensions.json

```diff
@@ -0,0 +1,53 @@
+[
+  {
+    "type": "console.flag/model",
+    "properties": {
+      "flag": "ODF_IBM_STORAGE",
+      "model": {
+        "group": "odf.ibm.com",
+        "version": "v1alpha1",
+        "kind": "FlashSystemCluster"
+      }
+    }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "ibmstoragesystem",
+      "insertBefore": "persistentvolumes",
+      "section": "storage",
```

Comment:
I am not sure if we need this nav as per designs ?

Reply:
No needed when GA. will remove it when odf-console is ready
This is for testing purpose when odf-console is not ready.